src/main/java/strategy/OperationStrategyImpl.java

```diff
@@ -0,0 +1,24 @@
+package strategy;
+
+import java.util.Map;
+import strategy.handler.BalanceOperationHandler;
+import strategy.handler.PurchaseOperationHandler;
+import strategy.handler.ReturnOperationHandler;
+import strategy.handler.SupplyOperationHandler;
+
+public class OperationStrategyImpl implements OperationStrategy {
+    private static final Map<String, OperationHandler> operationHalndlerMap;
+
+    static {
+        operationHalndlerMap = Map.of(
+                "b", new BalanceOperationHandler(),
+                "s", new SupplyOperationHandler(),
+                "p", new PurchaseOperationHandler(),
+                "r", new ReturnOperationHandler());
+    }
```

Comment:
It is better to use constructor
```suggestion
    public OperationStrategyImpl(Map<Operation, OperationHandler> operationHalndlerMap) {
        this. operationHalndlerMap = operationHalndlerMap;
    }
```